recipes/pystackreg/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pystackreg" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "4ab1c2cee5233a96faf29de47b8ace3892270f2334a7a3cb49e3369fb2db8fec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - numpy
+    - pip
+  run:
+    - python
+    - numpy
```

Comment:
I presume this is linking against numpy. If so, see https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Done, thanks for the hint!